okhttp/src/main/java/com/squareup/okhttp/Cache.java

```diff
@@ -270,6 +270,19 @@ private void abortQuietly(DiskLruCache.Editor editor) {
   }
 
   /**
+   * Initialize the cache. This will include reading the journal files from
```

Comment:
nit: also add a note that if the cache isn't initialized manually, it'll self-initialize on its first request?